packages/blocks/src/page-block/edgeless/edgeless-blocks-container.ts

```diff
@@ -166,11 +175,20 @@ export class EdgelessBlockContainer extends WithDisposable(LitElement) {
             : html`<affine-note-slicer
                 .edgelessPage=${edgeless}
               ></affine-note-slicer>`}
-          <edgeless-notes-container
-            .edgeless=${edgeless}
-            .notes=${sortedNotes}
-            .renderer=${renderModel.bind(this)}
-          ></edgeless-notes-container>
+          ${repeat(
+            blocks,
+            block => block.id,
+            (block, index) => {
+              const tag = literal`${unsafeStatic(
```

Comment:
The block flavour here comes from unreliable dynamic content, so using it transparently as HTML tag type seems to be unsafe.

Reply:
in this case, the flavour is one of note or image, so I think it's safe.